apps/explorer/src/index.css

```diff
@@ -15,30 +15,5 @@
 }
 
 body {
-    @apply m-0 font-sans text-offblack antialiased;
-}
-
-/*
- * Reset the font-family for elements that override their font-family by default.
- * This will no longer be needed once we use the Tailwind CSS Reset.
- */
-button,
-input,
-optgroup,
-select,
-textarea {
-    font-family: inherit;
-}
-
-input:focus,
-textarea:focus,
-select:focus {
-    outline: none;
-}
-
-/* Fix inconsistent font size rendering on iphone */
-@media screen and (max-device-width: 480px) {
-    body {
-        text-size-adjust: none;
-    }
+    @apply antialiased;
```

Comment:
Probably should leave `text-offblack` and `font-sans` for consistency.

Reply:
`font-sans` is added in preflight. text-color is not, I'll add back